src/Libraries/CoreNodes/FileSystem.cs

```diff
@@ -42,7 +42,7 @@ public static string AbsolutePath(string path, string hintPath = null)
                 var filepath = Path.Combine(parent, path);
                 //If hint path is null or file exists at this location return the computed path
                 //If hint path doesn't exist then the relative path might be for write operation.
-                if (File.Exists(filepath) || string.IsNullOrEmpty(hintPath) || !File.Exists(hintPath))
+                if (FileSystem.FileExists(filepath) || string.IsNullOrEmpty(hintPath) || !FileSystem.FileExists(hintPath))
```

Comment:
did you rename this with a rename occurrence or with a rename refactor? This is kind of ambiguous as there is a system.io.FileExists method which may be what this method used before - please verify this method was not calling system.io.fileExists.

Reply:
to clarify, did visual studio do a symbol based renaming or just a text equality based replace.

Reply:
Yeah I was also concerned I would accidentally rename system.io functions accidentally but tried to step through all the changes that were occurring in the renaming refactor to make sure.  The example above is not part of system.io but does call it internally.

![image](https://user-images.githubusercontent.com/13341935/32730316-b216bf08-c854-11e7-80ee-5196437f97f5.png)